src/components/memberPage/members.json

```diff
@@ -147,17 +147,16 @@
   ],
   "phd": [
     {
-      "name": "Che Ping Tsai",
-      "mail": "kingkazma0112305@gmail.com",
+      "name": "Leung Lok Him",
+      "mail": "lhleung@netdb.csie.ncku.edu.tw",
       "link": {
-        "github": "https://github.com/curry0622",
-        "linkedin": "https://www.linkedin.com/in/%E5%93%B2%E5%B9%B3-%E8%94%A1-9a97261a3/",
-        "homepage": "https://www.facebook.com/profile.php?id=100006457867592"
```

Comment:
Because we have prescribed the type of `link.homepage` and `link.linkedin` to `string | null`, you need to assign null to them like this:
```typescript
"link": {
        "github": "...",
        "linkedin": null,
        "homepage": null
}
```